windows-driver-docs-pr/install/inf-addservice-directive.md

```diff
@@ -406,14 +443,17 @@ AddReg=add-registry-section
 HKR,,BootFlags,0x00010003,0x14 ; CM_SERVICE_USB3_DISK_BOOT_LOAD|CM_SERVICE_USB_DISK_BOOT_LOAD
 ```
 
+> [!NOTE]
+> This syntax should be used only in Windows 11, builds older than 25381, for all others use **BootFlags** directive.
```

Comment:
I think the current text might imply that you can only use the AddReg in Windows 11 builds, but builds older than 25381.  The AddReg is valid in Win10 and below also, so suggesting a potential re-wording to make that clearer.
```suggestion
> This syntax should be used only in Windows versions and builds older than Windows 11 build 25381; for Windows 11 build 25381 and later, use the **BootFlags** directive.
```

Reply:
Fixed